plugins/net-forward.yaml

```diff
@@ -0,0 +1,47 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+command: ./kubectl-net_forward
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: net-forward
+spec:
+  version: "v1.0.2"       # required, must be in semver format, prefixed with "v"
+  platforms:
+  # specify installation script for linux 
+  - selector:             # a regular Kubernetes selector
+      matchLabels:
+        os: linux
+    # url for downloading the package archive:
+    uri: https://github.com/antitree/krew-tools/releases/download/0.0.2/krew-tools_v1.0.2.tar.gz
+    # sha256sum of the above archive file:
+    sha256: "2248c140613ce9a4796332744de6c37e82352929270bbd06b2ad6fde6e7c7f1c"
+    # copy the used files out of the zip archive, defaults to `[{from: "*", to: "."}]`
+    files:
+      - from: "net-forward/*"
```

Comment:
can you please also copy out the license?

Reply:
Apache license file included separately, others removed